bfe_module/bfe_module.go

```diff
@@ -44,18 +48,18 @@ type BfeModule interface {
 }
 
 // moduleMap holds mappings from mod_name to module.
-var moduleMap = make(map[string]BfeModule)
+var ModuleMap = make(map[string]BfeModule)
```

Comment:
the modification, too many codes to be affected. May add new function like `func GetModuleMap() map[string]BfeModule` will be better.